Convert translation data to SALT v2 format.ipynb

```diff
@@ -158,7 +158,18 @@
    "execution_count": null,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    unique_english_text = set()
</pre><p>If I'm understanding this correctly, this part of the code is to convert the entire SALT v1 dataset into the v2 format before adding it to the training data, and then the training is done with the v2 format, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SunbirdAI/parallel-text-EDA/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Training is now done on the v2 format. Processing is done using the saltv2/salt.py file from the github.com/sunbirdAI/nmt_training repo

Reply:
Noted